CHANGES.md

```diff
@@ -34,6 +34,7 @@
 - Fixed color palette outside viewport [Taiga #2715](https://tree.taiga.io/project/penpot/issue/2715)
 - Fixed missing translate string [Taiga #2780](https://tree.taiga.io/project/penpot/issue/2780)
 - Fixed handoff shadow type text [Taiga #2717](https://tree.taiga.io/project/penpot/issue/2717)
+- Fixed cannot align objects inside a group but not inside a frame [Taiga #2762](https://tree.taiga.io/project/penpot/issue/2762)
```

Comment:
I think I'm not clearly understand this changelog entry

Reply:
it's explained in the issue...